packages/block-editor/src/components/media-upload/index.native.js

```diff
@@ -36,6 +39,8 @@ export const OPTION_TAKE_VIDEO = __( 'Take a Video' );
 export const OPTION_TAKE_PHOTO = __( 'Take a Photo' );
 export const OPTION_TAKE_PHOTO_OR_VIDEO = __( 'Take a Photo or Video' );
 
+const URL_MEDIA_SOURCE = 'URL';
```

Comment:
The `mediaSources` being used throughout this component lives within the bridge. I opted not to put this there since inserting from the URL is not bridge-related.